src/app/pages/page-not-found/page-not-found.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit, ChangeDetectionStrategy } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-page-not-found',
+  templateUrl: './page-not-found.component.html',
+  styleUrls: ['./page-not-found.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PageNotFoundComponent implements OnInit {
+  path: string
+
+  constructor(private route: ActivatedRoute) {
+    this.path = "\"/" + this.route.snapshot.url.join('/') + "\"";
```

Comment:
```suggestion
constructor(public route: ActivatedRoute) {
```

I think we can just use the observable from `ActivatedRoute` right in the template (see other suggestion).